ci-operator/step-registry/openshift-qe/ocp-61589-max-nodes-network-expansion/openshift-qe-ocp-61589-max-nodes-network-expansion-commands.sh

```
@@ -0,0 +1,361 @@
+
+# Function to wait for nodes to be ready
+wait_for_nodes() {
+    local target_ready=$1
+    local timeout=$2
+    echo "Waiting for $target_ready nodes to be ready (timeout: $timeout)"
+    
+    local start_time
+    start_time=$(date +%s)
+    local timeout_seconds
+    if [[ "$timeout" =~ ([0-9]+)h ]]; then
+        timeout_seconds=$((${BASH_REMATCH[1]} * 3600))
```

Shellcheck (0.11.0)

[style] $/${} is unnecessary on arithmetic variables.

(SC2004)